.github/workflows/check-links-cron.yaml

```diff
@@ -0,0 +1,69 @@
+name: Periodically Check Documentation Links
+
+on:
+  schedule:
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/check-links-cron.yaml'
+
+permissions: read-all
+
+jobs:
+  links-checker:
+    name: Validate & Build HTML
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          persist-credentials: false 
+      - name: Build HTML
+        run: |
+          set -ex
+          # chmod +x ./Documentation/build.sh
+          # ./Documentation/build.sh

```

Comment:
We probably don't need to build the docs?

Reply:
Hey sorry forgot to make it a draft. Was testing a bunch of things to make sure workflow works properly. Will ping you once I am done with the changes. 
Should be done by tomorrow